plugins/systems/attack/Attack.hpp

```diff
@@ -52,6 +52,8 @@ namespace Systems {
                 auto &reg = engine.getRegistry();
                 ECS::Entity projectile = reg.createEntity();
 
+                std::cerr << "Created new projectile with id: " << projectile << std::endl;
```

Comment:
Why `cerr` and not `cout` ?

Reply:
We currently have 2 types of debug in our Game, normal debug that shows the different components at runtime for each iteration, and debug that prints errors and whats happening behind the scene.

For that, we decided to put this first type of debug in cout and the second type in cerr in order to be able to separate both and be able to check one or another. (Using redirections ofc)

Reply:
thx for the explanations 👍